org.BleuBleu.FamiStudio.yml

```diff
@@ -0,0 +1,65 @@
+app-id: org.BleuBleu.FamiStudio
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
+command: famistudio.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+# Read and write projects
+  - --filesystem=home
+# For the Mono runtime
+  - --env=MONO_GAC_PREFIX=${FLATPAK_DEST}
+# Midi keyboard input
+  - --device=all
+modules:
```

Comment:
Not critical but should be reformatted to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide.

Reply:
Sorry, what do you want me to change exactly?

Reply:
Modules should be separated by new lines, mostly.